Fluid/Filters/ColorFilters.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Drawing;
+using System.Text.RegularExpressions;
```

Comment:
No regular expressions in Fluid. I am alergic.

Reply:
I hesitate before I use it .. time to remove ...

Reply:
AFAIK Regular Expression is not good for PERF but makes the LIVE easier ;) so hope the latest changes makes you happy